dataverse-persistence/src/main/resources/Bundle_en.properties

```diff
@@ -3036,14 +3036,15 @@ failed.login.cause.duplicated.email=Your email address provided by your institut
 failed.login.cause.authentication.error=Authentication error. If the problem persists, please contact repository support.
 
 uningest.page.title=Uningest
-uningest.page.info=Lorem ipsum dolor sit amet\u2026
+uningest.page.info=<strong>Uningest file</strong> - restore a file to its original form and remove its extracted metadata. Uningest is only possible for files in xlsx, csv and tsv original formats, which have not yet been published in any version of the dataset.
```

Comment:
Here and in the polish version: I would recommend to replace `-` with `–` (`\u2013`).